R/cf.R

```diff
@@ -329,7 +360,7 @@ bootstrap.cf <- function(cf, boot.R=400, boot.l=2, seed=1234, sim="geom", endcor
   return(invisible(cf))
 }
 
-jackknife.cf <- function(cf, boot.l=2) {
+jackknife.cf <- function(cf, boot.l = 2) {
```

Comment:
can we default to "leave out 1" ?

Reply:
Of course, this was just the default before. It is changed now.